src/stochastic_closures.jl

```diff
@@ -1,5 +1,5 @@
 import Distributions
-import DifferentialEquations: SOSRI
+import StochasticDiffEq: SDEProblem, SOSRI, solve
```

Comment:
```suggestion
import StochasticDiffEq
const SDEQ = StochasticDiffEq
```
Can we move this to `src/TurbulenceConvection.jl` and then just use `SDEQ.SDEProblem`/ `SDEQ.SOSRI` / `SDEQ.solve`?